feature_development.md

```diff
@@ -0,0 +1,26 @@
+# Feature Development
+
+The feature development playbook is for tracking progress in feature
+development. A Software Engineer or Product Manager might want to track the
+velocity of implemented features in order to capture additional value. This
+information can be used as track record as well as community update.
+
```

Comment:
Our focus is the founder (not product manager or software engineer). 

Could say - For founders who want to establish a public track record.

Reply:
My thoughts were these. If the founder is alone, the founder obtains all of these roles. If the founder is not alone there might be a person for that role. So all of this is not literal. Playbooks are inspirational. You need to use your own brain still. If we rewrite it, I can make the counter argument that founders would not obtain these roles themselves all the time which would render the wording inaccurate again. I want to draw a fuzzy picture so that people realize that playbooks are not black and white.

Reply:
Founders is generic for any primary creator of a venture. It encompasses any sub-roles but software engineer or product manager does not imply founder.

Put another way - for every founder-engineer there are 10,000 engineers and they view the world differently. In our outward facing language we want engineers that identify as founders (first) rather than just engineers.